tasks/NASA Moon survival.md

```diff
@@ -0,0 +1,45 @@
+# Summary
+This task is the NASA moon survival test. This requires subjects to imagine themselves crash landed on the moon 200 miles from base. All but 15 pieces of equipment have been destroyed. The surviving items are to be ranked in order of their declining contribution to survival on the long trek to safety. 
+
+# References
+Main paper: https://www-sciencedirect-com.proxy.library.upenn.edu/science/article/pii/0030507383901447?via%3Dihub
+
+NASA task: Hall, J., & Watson, W. H. The effects of a normative intervention on group decision making. Human Relations, 1970, 23, 299-317
+
+# Stimuli
+## The visual components
+![NASA](/images/NASA.png)
+
+
+# Procedure
+## Steps
+Interacting groups:
+> Subjects first solved the NASA moon problem as individuals
+> There were then randomly allocated to groups and developed a team solution
+> Ten minutes were allowed for the first phase and 30 for the second
+
+Simulation groups:
+> Groups were constructed with a computer routine
+> Individuals were drawn at random to form groups of 1 to 7 members
+> Three simulations were run
+> > 1. Groups were formed with members drawn at random from the total distribution of NASA scores. 200 teams were sampled for each size level
+> > 2. Groups were formed within two ability strate; above median ability and below median ability. 200 teams were sampled for each size and ability level.
+> > 3. Groups were formed within size specific ability bands. Fifty groups were simulated for each size and ability level.
+
+> AM NOT SURE WHAT TO SAY ABOUT UNIT AND NONUNIT WEIGHT
```

Comment:
> To test Proposition 2, groups were assigned to unit and nonunit weight categories as follows. First, using the member solutions, artificial w= 0 and w = 2 group solutions were generated. Second, deviation scores, that is, the sums of absolute differences between actual and artificial groups were computed. Groups which deviated less from the to w= 0 strategy than the w = 2 strategy were assigned to the unit weight category, and vice versa.

From what I understand, basically they assigned weights to the groups individual scores (i.e. each member gets assigned weight 0 or weight 2), and averaged them to make artificial group scores.  They then computed standard deviations between the weighted scores and the "actual group" (not sure what this means in this case, could be an unweighted average or perhaps the actual outcome from the group) and put them in a weight category based on which one was closer to reflecting the score.